quotes/quotes.js

```diff
@@ -121,7 +128,7 @@ function createQuote(data, roles, number, quote) {
             tagSpan.innerHTML = userdata.tag
         }
 
-        content.appendChild(createPopup(user, roles, userdata))
+        nameDiv.onfocus = () => document.getElementById("popup-container-" + user)?.focus({preventScroll:true})
```

Comment:
What does this do, pray tell?

Reply:
So the current way the popups work is based on the containing div gaining focused.
With the deduplication this still works but we have a new single container and need to be able to focus that container from multiple places, we can't use pure css for this anymore (afaik there may be some sort of trick that would work or a redesign).

This adds a `on focus` listener to what were the multiple old containers that insteads forwards the focus to the new single container for a user instead.

This single container gets generated earlier in the script and only once per user.

Reply:
Good good, possibly consider adding a brief comment to that effect, like `// Forward focus events to popup container for user`.